From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@guyburstein__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

carolinacmoravia committed:
M5236: Incomplete sentence: "To fully sign a user out of a web application, your app should end its own session with the user (usually by clearing a token cache or dropping cookies), and then redirect the browser to"

@dastrock, 
It seems that the sentence is incomplete. Could you review this and reply, please?
Many thanks.

puicchan committed:
Update install-jenkins-solution-template.md

Clarified requirement for using existing resource group. Iif using an existing resource group, it must be empty.

Commit:
accpanel01

tamram committed:
rendering service overview

Commit:
Added Article for Service Remoting Performance Counters

Commit:
Minor Editing to the Content

tamram committed:
rendering service overview

tamram committed:
rendering service overview

tamram committed:
rendering service overview

tamram committed:
rendering service overview

sideye committed:
Fix typo

sideye committed:
Capitalize connection string replacement

For increased noticeability

sideye committed:
PuTTy for windows screenshot

sideye committed:
Separate SSH setup instructions by OS

Add PuTTy Screenshot

sideye committed:
Update iot-hub-raspberry-pi-kit-node-get-started.md

sideye committed:
Fix spacing

sideye committed:
Fix spacing

sideye committed:
Fix order in "What you Do" section

sideye committed:
Fix Raspbian download description

sideye committed:
Add units for interval description

sideye committed:
Add recommendation for C2D D2C messaging in "next steps"

sideye committed:
Fix next steps description

sideye committed:
Add link to skip section for users w/o BME sensor

sideye committed:
Fix link to anchor

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0724

kgremban committed:
App Proxy QP week 3

sideye committed:
Update iot-hub-raspberry-pi-kit-c-get-started.md

sideye committed:
Add IP address section

sideye committed:
Update SSH connection method

sideye committed:
Update RPi C in accordance with Node

kgremban committed:
Numbered H4s

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0724

tamram committed:
rendering service overview

tamram committed:
rendering service overview

tamram committed:
rendering service overview

tamram committed:
rendering service overview

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0724

tamram committed:
rendering service overview

tamram committed:
rendering service overview

tamram committed:
rendering service overview

tamram committed:
rendering service overview

Commit:
updates for freshness

MGoedtel committed:
Updated upgrade article with new image and support information

cherylmc committed:
new article

cherylmc committed:
fix merge conflict

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into routefilter2

cherylmc committed:
update to add link

cherylmc committed:
update format

cherylmc committed:
adjust dates for warning

markgalioto committed:
adding the back up system State article

kgremban committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into jul18_appproxyqp3

kgremban committed:
Updated metadata for App Proxy oldportal docs

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-0724

tamram committed:
rendering service overview

sideye committed:
Remove step 1 in line 205

priyaravi20 committed:
Update GetStartedJSWebsockets.md

v-shils committed:
Merge pull request #18253 from tamram/tamram-0724

rendering service overview

markgalioto committed:
checking formatting of inline code

Commit:
Minor Fixes to the name and content as suggested by reviewer

Commit:
Merge pull request #18370 from priyaravi20/patch-6

Update GetStartedJSWebsockets.md

tfitzmac committed:
deleted deploy topic

v-shils committed:
Merge pull request #18315 from sideye/patch-1

Fix and update page with Solutions from User Testing

Commit:
Merge pull request #17906 from puicchan/master

Update install-jenkins-solution-template.md

markgalioto committed:
fixing broken tables

kgremban committed:
Finished network topo changes (except art)

cherylmc committed:
update

kgremban committed:
Merge branch 'jul18_appproxyqp3' of https://github.com/kgremban/azure-docs-pr into jul18_appproxyqp3

Commit:
cba02

carolinacmoravia committed:
Update active-directory-v2-protocols-implicit.md

Ja-Dunn committed:
Merge pull request #18371 from tfitzmac/0731delete

deleted deploy topic

Commit:
Merge pull request #18375 from MarkusVi/cba02

cba02

Commit:
Merge pull request #14750 from carolinacmoravia/patch-219

M5236: Incomplete sentence: "To fully sign a user out of a web applic…

CarlRabeler committed:
Update sql-database-automated-backups.md

Ja-Dunn committed:
Merge pull request #18356 from cherylmc/routefilter2

new article

Ja-Dunn committed:
Merge pull request #18353 from MGoedtel/UpdateLANewQuery7312017

Updated upgrade article with new image and support information

Commit:
freshness04

Ja-Dunn committed:
Merge pull request #18351 from Blackmist/fresh2

updates for freshness

carolinacmoravia committed:
M4761: Link not working: [Premium Storage scalability and performance… #11978

@ramankum,
Please, review if the proposed file change is correct. Thank you.

Commit:
accpanel01

Commit:
accpanel01

Ja-Dunn committed:
Merge pull request #18314 from kgremban/jul18_appproxyqp3

App Proxy QP week 3

Commit:
Merge pull request #18377 from MarkusVi/freshness04

freshness04

markgalioto committed:
updated the TOC to add the new file backup-azure-system-state

Commit:
Merge pull request #18376 from CarlRabeler/patch-312

Update sql-database-automated-backups.md

tamram committed:
rendering service - updates

cherylmc committed:
update date

Commit:
Merge pull request #18384 from cherylmc/quickfix

update date

tamram committed:
rendering service - updates

TomShinder committed:
Update security-azure-log-integration-get-started.md

updating SIEM vendor statement in Next Steps.

kgremban committed:
App Proxy QP week 3 - pt 2, UAG/TMG

Commit:
Merge pull request #18380 from carolinacmoravia/patch-294

M4761: Link not working: [Premium Storage scalability and performance…

Commit:
Merge pull request #18368 from kgremban/jul31_appproxyoldportal

Updated metadata for App Proxy oldportal docs

ajlam committed:
Update JDBC download center page

Commit:
Merge pull request #18386 from TomShinder/patch-21

Update security-azure-log-integration-get-started.md

Commit:
Merge pull request #18390 from ajlam/patch-3

Update JDBC download center page

tfitzmac committed:
add cloud shell deploy

Commit:
Merge pull request #18385 from tamram/tamram-0731

rendering service - updates

Ja-Dunn committed:
Merge pull request #18160 from MarkusVi/accpanel01

accpanel01

Ja-Dunn committed:
Merge pull request #18272 from suchiagicha/master

Adding Article for Service Remoting Performance Counters

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Ja-Dunn committed:
Merge pull request #18359 from markgalioto/Az7-31

adding the back up system State article

Ja-Dunn committed:
Merge pull request #18392 from tfitzmac/0731shell

add cloud shell deploy

v-alje committed:
Merge pull request #18395 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #18387 from kgremban/jul31_appproxyqp3.2

App Proxy QP week 3 - pt 2, UAG/TMG
